src/components/links/link/index.tsx

```diff
@@ -22,11 +23,13 @@ const Link: React.FC<LinkProps> = ({
   ...rest
 }: LinkProps) => {
   const href = rest.disabled ? '' : url
-  return (
-    <S.Link to={href} size={size} {...rest}>
-      {children}
-    </S.Link>
-  )
+  const Component = isExternalURL(href) ? S.ExternalLink : S.InternalLink
+  const props = {
+    size,
+    [isExternalURL(href) ? 'href' : 'to']: href,
+    ...rest,
+  }
+  return <Component {...props}>{children}</Component>
```

Comment:
Nešlo by namísto ternárních operátorů použít něco jako následující forma?  

```typescript jsx
if (isExternalUrl(href) {
   return <S.ExternalLink href={href} {...props} />
}

return <S.InternalLink to={href} {...props} />
```

Reply:
nemam rad copy/pase ve `href={href} {...props}` a `to={href} {...props}`